src/library/styles/pxToEm.js

```diff
@@ -4,8 +4,20 @@
  */
 import fontSize_base from '../themes/fontSizeBase';
 
+const justNumbersOrEndsWithPx = (s: string) => s.match(/^[\d.]+(px)?$/);
```

Comment:
* Regex should be extracted to const so it does not get recreated each time this is called.
```js
const REGEX_NUM_OR_PX = /^[\d.]+(px)?$/;
```
* Can use `test` rather than `match` if you are not doing anything with the matches.
* Actually this function is really unnecessary as the `test` can simply be inlined.
```js
REGEX_NUM_OR_PX.test(value)
```